test/Prototypes/FloatingPoint.swift

```diff
@@ -858,7 +858,7 @@ public protocol FloatingPointInterchange: FloatingPoint {
 
 extension FloatingPointInterchange {
   public init(littleEndian encoding: BitPattern) {
-#if arch(i386) || arch(x86_64) || arch(arm) || arch(arm64) || arch(powerpc64le)
+#if arch(i386) || arch(x86_64) || arch(arm) || arch(arm64) || arch(powerpc64le) || arch(s390x)
```

Comment:
Feel free to submit changes to this file as a separate PR, it can be merged immediately.